src/Lecture1.hs

```diff
@@ -123,4 +130,15 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Integer -> [Integer] -> [Char]
+lowerAndGreater n list = 
+    let greater = 0
+        lower = 0
+    in comp list greater lower
+        where
+            comp :: [Integer] -> Integer -> Integer -> [Char]
+            comp list greater lower
```

Comment:
I see GHC warning about shadowing variable `list` in the `comp` function. To avoid this warning, you can name these two variables differently.

Reply:
Okay, mate. Thanks for all the tips and pointers. 
Appreciate it.